README.md

```diff
@@ -314,52 +315,95 @@ module.exports = {
 
-Type: `Array<string>`. Absolute paths, please. Default: `[]`.
+### devServerHistoryFallback
 
-An array of filenames pointing to stylesheets that you want to include in your site.
+Type: `boolean`. Default: `false`.
 
-These will be processed by PostCSS with [Autoprefixer](https://github.com/postcss/autoprefixer) and any other [postcssPlugins](#postcssplugins) that you specify; concatenated in the order specified; and added to the `<head>` of your document.
+Set to `true` if you want to use HTML5 History for client-side routing (as opposed to hash routing). This configures the development server to fall back to `index.html` when you request nested routes.
 
-Assets referenced by your stylesheets will be hashed and copied to the [outputDirectory](#outputdirectory), unless they are absolute URLs.
+This is `false` by default because it should only be *intentionally* turned on, when you know you're going to configure your server to allow for HTML5 History—powered client-side routing.
 
-### vendorModules
+### htmlSource
 
-Type: `Array<string>`. Default: `[]`.
+Type: `string`. Absolute path, please. Default: `${project-directory}/src/html.js`.
 
-Identifiers of npm modules that you want to be added to the vendor bundle.
-The purpose of the vendor bundle is to deliberately group dependencies that change relatively infrequently — so this bundle will stay cached for longer than the others.
+The path to your HTML source file. For more information, read [Defining your HTML](#defining-your-html).
 
-By default, the vendor bundle includes `react` and `react-dom`.
+### jsEntry
 
-### devServerHistoryFallback
+Type: `string`. Absolute path, please. Default: `${project-directory}/src/index.js`.
 
-Type: `boolean`. Default: `false`.
+The entry JS file for your app. Typically this is the file where you'll use `react-dom` to render your app on an element.
 
-Set to `true` if you want to use HTML5 History for client-side routing (as opposed to hash routing). This configures the development server to fall back to `index.html` when you request nested routes.
+In the default value, `project-directory` refers to the directory of your `underreact.config.js` file, or the current working directory.
 
-This is `false` by default because it should only be *intentionally* turned on, when you know you're going to configure your server to allow for HTML5 History—powered client-side routing.
+### outputDirectory
 
-### htmlSource
+Type `string`. Absolute path, please. Default: `${project-directory}/_underreact-site/`.
 
-Type: `string`. Absolute path, please. Default: `${project-directory}/src/html.js`.
+The directory where files should be written.
 
-The path to your HTML source file. For more information, read [Defining your HTML](#defining-your-html).
+You'll want to ignore this directory with `.gitignore`, `.eslintignore`, etc.
+
+In the default value, `project-directory` refers to the directory of your `underreact.config.js` file, or the current working directory.
```

Comment:
"or the current working directory" when?

Reply:
I have removed this line.